File: sgifd-backend/src/main/java/com/sgifdbackend/demo/parametrage/entites/ZoneLocalite.java
package com.sgifdbackend.demo.parametrage.entites;

import com.sgifdbackend.demo.entitiesBase.EntityBaseBean;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Table(name = "t_ZoneLocalite")
public class ZoneLocalite extends EntityBaseBean implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String libelle;

    @ManyToOne
    private Arrondissement arrondissement;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getLibelle() {
        return libelle;
    }

    public void setLibelle(String libelle) {
        this.libelle = libelle;
    }

	public Arrondissement getArrondissement() {
		return arrondissement;
	}

	public void setArrondissement(Arrondissement arrondissement) {
		this.arrondissement = arrondissement;
	}
    
}

File: sgifd-frontend/src/app/models/ZoneLocalite.ts
import { Arrondissement } from './Arrondissement';

export class ZoneLocalite {
  id: number;
  libelle: string;
  createBy:	string;
  deleteBy:	string;
  arrondissement: Arrondissement;

  constructor(id: number, libelle: string, createBy: string, deleteBy: string , arrondissement: Arrondissement) {
    this.id = id;
    this.libelle = libelle;
    this.createBy = createBy;
    this.deleteBy = deleteBy;
    this.arrondissement = arrondissement;
  }
}
